### PR TITLE
Remove unused usings.

### PR DESCRIPTION
**Pull request by HeroMaxPower**
Removed a large amount of unused usings.

### CODE REVIEW

File: src/Microsoft.TestPlatform.ObjectModel/Adapter/TestPlatformFormatException.cs [hunk 1]
```diff
@@ -4,11 +4,9 @@
 namespace Microsoft.VisualStudio.TestPlatform.ObjectModel.Adapter
 {
     using System;
-    using System.Runtime.Serialization;
 #if NET451
-using System.Security.Permissions;
+    using System.Runtime.Serialization;
```
Comment (nohwnd) [inline src/Microsoft.TestPlatform.ObjectModel/Adapter/TestPlatformFormatException.cs]: weird change, the namespace that was outside of the compiler directive was moved inside, please revert.

File: src/Microsoft.TestPlatform.Utilities/XmlUtilities.cs [hunk 2]
```diff
@@ -5,7 +5,9 @@ namespace Microsoft.VisualStudio.TestPlatform.Utilities
 {
     using System;
     using System.Diagnostics.CodeAnalysis;
+#if NET451
```
Comment (nohwnd) [inline src/Microsoft.TestPlatform.Utilities/XmlUtilities.cs]: why? Is this suggested by some tool?

File: test/Microsoft.TestPlatform.Utilities.UnitTests/InferRunSettingsHelperTests.cs [hunk 3]
```diff
@@ -1,23 +1,18 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
-namespace Microsoft.TestPlatform.Utilities.UnitTests
-{
+namespace Microsoft.TestPlatform.Utilities.UnitTests {
```
Comment (nohwnd) [inline test/Microsoft.TestPlatform.Utilities.UnitTests/InferRunSettingsHelperTests.cs]: move the { to the next line please.

**Review by nohwnd — CHANGES_REQUESTED** [review]

File: src/Microsoft.TestPlatform.Utilities/XmlUtilities.cs (hunk 2 above)
Comment (HeroMaxPower) [reply src/Microsoft.TestPlatform.Utilities/XmlUtilities.cs]: SecurityElement.Escape(innerXml); uses this library only so a tool SonarQube suggested to create a block for NET451 the using doesn't apply to NETCORE2.0. im not sure if this is an anti pattern though. Let me know if you want to revert it.

File: test/Microsoft.TestPlatform.Utilities.UnitTests/InferRunSettingsHelperTests.cs (hunk 3 above)
Comment (HeroMaxPower) [reply test/Microsoft.TestPlatform.Utilities.UnitTests/InferRunSettingsHelperTests.cs]: done

File: src/Microsoft.TestPlatform.ObjectModel/Adapter/TestPlatformFormatException.cs (hunk 1 above)
Comment (HeroMaxPower) [reply src/Microsoft.TestPlatform.ObjectModel/Adapter/TestPlatformFormatException.cs]: reverted

**Review by nohwnd — APPROVED** [review]

**Pull request merged**